packages/cashier/package.json

```diff
@@ -74,7 +74,7 @@
         "@testing-library/react": "^12.0.0",
         "@types/react": "^18.0.7",
         "@types/react-dom": "^18.0.0",
-        "@testing-library/user-event": "^13.2.1",
+        "@testing-library/user-event": "^13.5.0",
```

Comment:
the same question here about adding extra testing features

Reply:
@yauheni-kryzhyk-deriv, the `"@testing-library/user-event": "^13.2.1",` was added in [this](https://github.com/binary-com/deriv-app/pull/6339) PR, I just upgraded it to match the one that was added to `POO`